[25.05] simplex-chat-desktop: 6.3.3 -> 6.4.5

TerryGarcia opened the pull request:
<!--
^ Please summarise the changes you have done and explain why they are necessary here ^

For package updates please link to a changelog or describe changes, this helps your fellow maintainers discover breaking updates.
For new packages please briefly describe the package or provide a link to its homepage.
-->

Manual backport of https://github.com/NixOS/nixpkgs/pull/441718


## Things done

<!-- Please check what applies. Note that these are not hard requirements but merely serve as information for reviewers. -->

- Built on platform:
  - [ ] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- Tested, as applicable:
  - [ ] [NixOS tests] in [nixos/tests].
  - [ ] [Package tests] at `passthru.tests`.
  - [ ] Tests in [lib/tests] or [pkgs/test] for functions and "core" functionality.
- [ ] Ran `nixpkgs-review` on this PR. See [nixpkgs-review usage].
- [ ] Tested basic functionality of all binary files, usually in `./result/bin/`.
- Nixpkgs Release Notes
  - [ ] Package update: when the change is major or breaking.
- NixOS Release Notes
  - [ ] Module addition: when adding a new NixOS module.
  - [ ] Module update: when the change is significant.
- [ ] Fits [CONTRIBUTING.md], [pkgs/README.md], [maintainers/README.md] and other READMEs.

[NixOS tests]: https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests
[Package tests]: https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-tests
[nixpkgs-review usage]: https://github.com/Mic92/nixpkgs-review#usage

[CONTRIBUTING.md]: https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md
[lib/tests]: https://github.com/NixOS/nixpkgs/blob/master/lib/tests
[maintainers/README.md]: https://github.com/NixOS/nixpkgs/blob/master/maintainers/README.md
[nixos/tests]: https://github.com/NixOS/nixpkgs/blob/master/nixos/tests
[pkgs/README.md]: https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md
[pkgs/test]: https://github.com/NixOS/nixpkgs/blob/master/pkgs/test

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

mdaniels5757 commented:
## `nixpkgs-review` result

Generated using [`nixpkgs-review-gha`](https://github.com/Defelo/nixpkgs-review-gha)

Command: `nixpkgs-review pr 448168`
Commit: [`ad736783e7ddd0d3cb2c3e318490b9b57d9fd7d1`](https://github.com/NixOS/nixpkgs/commit/ad736783e7ddd0d3cb2c3e318490b9b57d9fd7d1) ([subsequent changes](https://github.com/NixOS/nixpkgs/compare/ad736783e7ddd0d3cb2c3e318490b9b57d9fd7d1..pull/448168/head))
Merge: [`77f2b187c620ddf55381caf418071d3fbd7013a4`](https://github.com/NixOS/nixpkgs/commit/77f2b187c620ddf55381caf418071d3fbd7013a4)

Logs: https://github.com/mdaniels5757/nixpkgs-review-gha/actions/runs/18249754415


---
### `x86_64-linux`
<details>
  <summary>:white_check_mark: 1 package built:</summary>
  <ul>
    <li>simplex-chat-desktop</li>
  </ul>
</details>


---
### `aarch64-linux`
:white_check_mark: *No rebuilds*


---
### `x86_64-darwin`
:white_check_mark: *No rebuilds*


---
### `aarch64-darwin`
:white_check_mark: *No rebuilds*

mdaniels5757 reviewed (approved):
LGTM.

Just an FYI: you could have avoided the cherry-pick warning by cherry-picking all of the version bumps (i.e. `git cherry-pick -x a11eec51b56c3318abff15ef8d557e8127014b6b (rest of bumps here...) 88c5e8e63d334a703b8867179088c9be8d8baa32`)

HeitorAugustoLN committed:
cosmic-greeter: add missing cosmic-randr and orca dependency

The path to the orca screen reader was hardcoded to /usr/bin/orca, which prevents it from being executed in a NixOS environment.

This change adds orca as a dependency and patches the source code to use the Nix-provided executable path, allowing the greeter to launch it correctly.

Also adds cosmic-randr to PATH.

HeitorAugustoLN committed:
nixos/cosmic-greeter: install cosmic-comp, cosmic-greeter by default

HeitorAugustoLN committed:
nixos/cosmic-greeter: create `/run/cosmic-greeter` directory

Also uses the right home mode for `cosmic-greeter` user

HeitorAugustoLN committed:
nixos/cosmic-greeter: add video group to user

This change aligns the NixOS module with the permissions configured by the upstream project.

uninsane committed:
superTuxKart: fix build against cmake 4

wrvsrx committed:
transmission_3: fix build with cmake 4

vbgl committed:
minisat: fix build with CMake 4

vbgl committed:
abc-verifier: unstable-2023-10-13 → 0.55

cyclic-pentane committed:
soapyremote: backport cmake < 3.5.0 fix

cyclic-pentane committed:
soapyhackrf: backport cmake < 3.5 fix

cyclic-pentane committed:
hackrf: backport cmake < 3.5 fix

Backports the hackrf commit that raises the minimum cmake version to
3.10, so the newest version of cmake doesn't complain about having
removed cmake < 3.5 compatibility anymore.

cyclic-pentane committed:
libbladeRF: 2.5.0 -> 2025.10

cyclic-pentane committed:
soapybladerf: backport cmake < 3.5.0 fix

jtojnar committed:
cog: drop (#450076)

jtojnar committed:
cinny-desktop: mark broken (#450214)

GaetanLepage committed:
vimPlugins.Otree-nvim: init at 2025-09-18 (#450361)

GaetanLepage committed:
python3Packages.ripser: disable failing tests on Darwin (#448967)

GaetanLepage committed:
vimPlugins.xeno-nvim: init at 2025-08-27 (#450364)

Sigmanificient committed:
python3Packages.pyoxigraph: 0.4.11 -> 0.5.0 (#444596)

Sigmanificient committed:
arti: 1.5.0 -> 1.6.0 (#449948)

r-ryantm committed:
tmuxai: 1.1.1 -> 1.1.2

fabaff committed:
python3Packages.airos: 0.5.4 -> 0.5.5 (#450473)

fabaff committed:
python3Packages.volvocarsapi: 0.4.2 -> 0.4.3 (#450475)

fabaff committed:
fosrl-newt: 1.5.1 -> 1.5.2 (#450426)

fabaff committed:
python3Packages.nicegui: 2.24.2 -> 3.0.3 (#450414)

fabaff committed:
python313Packages.yara-x: modernize

Updated changelog URL to reflect correct version tag.

fabaff committed:
python3Packages.chromadb: 1.1.0 -> 1.1.1 (#450345)

r-ryantm committed:
python3Packages.langsmith: 0.4.31 -> 0.4.34

jopejoe1 committed:
teamviewer: 15.69.4 -> 15.70.4 (#449239)

mweinelt committed:
python311: 3.11.13 -> 3.11.14; python310: 3.10.18 -> 3.10.19 (#450480)

happysalada committed:
cargo-mobile2: 0.20.7 -> 0.21.0 (#450442)

fabaff committed:
python313Packages.gto: 1.8.0 -> 1.9.0 (#450382)

fabaff committed:
python313Packages.channels-redis: modernize

Ratakor committed:
nixos/pmount: init module

r-ryantm committed:
protoc-gen-grpc-java: 1.75.0 -> 1.76.0

h7x4 committed:
pmount: Fix LUKS integration & update patches (#433847)

h7x4 committed:
nixos/pmount: init module (#439935)

r-ryantm committed:
upsies: 2025.09.20 -> 2025.10.09

r-ryantm committed:
mpv-handler: 0.3.16 -> 0.4.1

jopejoe1 committed:
tealdeer: 1.7.2 -> 1.8.0 (#448381)

r-ryantm committed:
olympus-unwrapped: 25.09.30.01 -> 25.10.08.01

jopejoe1 committed:
xtf: 0-unstable-2025-07-26 -> 0-unstable-2025-10-02 (#448278)

aanderse committed:
kodi.packages.kodi-platform: 17.1 -> 20250416

fabaff committed:
python3Packages.channels-redis: 4.2.1 -> 4.3.0 (#450483)

h7x4 committed:
yggdrasil-jumper: 0.3.1 -> 0.4.1, update module (#441480)

Sigmanificient committed:
kaitai-struct-compiler: use jre instead of jdk8 (#424577)

jopejoe1 committed:
cri-o-unwrapped: 1.34.0 -> 1.34.1 (#447870)

nixpkgs-merge-bot[bot] committed:
stylelint: 16.24.0 -> 16.25.0 (#448258)

aanderse committed:
kodi.packages.kodi-platform: 17.1 -> 20250416 (#450503)

fabaff committed:
python3Packages.yara-x: 1.7.1 -> 1.8.1 (#450380)

r-ryantm committed:
qcal: 0.9.2 -> 0.9.3

r-ryantm committed:
qcard: 0.7.1 -> 0.7.2

aanderse committed:
kodi: drop unused systemd dependency (#450452)

Sigmanificient committed:
olympus-unwrapped: 25.09.30.01 -> 25.10.08.01 (#450506)

nixpkgs-merge-bot[bot] committed:
mpv-handler: 0.3.16 -> 0.4.1 (#450504)

nixpkgs-merge-bot[bot] committed:
siyuan: 3.3.3 -> 3.3.4 (#450203)

siraben committed:
pot: add alias (#450268)

Sigmanificient committed:
uni-sync: 0.2.0 -> 0.3.1 (#445816)

tebriel committed:
audiobookshelf: 2.29.0 -> 2.30.0

changelog: https://github.com/advplyr/audiobookshelf/releases/tag/v2.30.0

nixpkgs-merge-bot[bot] committed:
komga: 1.23.4 -> 1.23.5 (#449909)

Sigmanificient committed:
ipfs-cluster: fix build (#449514)

Sigmanificient committed:
hyprviz: init at 0.6.1 (#442448)

Sigmanificient committed:
nftables: fix static build (#441065)

kira-bruneau committed:
zynaddsubfx: add cmakeFlags for CMake >= 3.5 (#450012)

typedrat committed:
imgbrd-grabber: fix build with CMake 4

r-ryantm committed:
openseachest: 25.05.2 -> 25.05.3

booxter committed:
nftables: fix build failure

r-ryantm committed:
rime-moegirl: 20250909 -> 20251009

r-ryantm committed:
laravel: 5.19.0 -> 5.20.0

K900 committed:
nftables: fix build failure (#450524)

r-ryantm committed:
python3Packages.gdsfactory: 9.17.0 -> 9.18.1

nixpkgs-merge-bot[bot] committed:
rime-moegirl: 20250909 -> 20251009 (#450526)

drupol committed:
feyngame: fix source reproducibility (#450478)

nixpkgs-merge-bot[bot] committed:
gollama: v1.37.2 -> v1.37.3 (#449690)

afh committed:
sonic: fix install step on Darwin (#449620)

dotlambda committed:
libdeltachat: 2.17.0 -> 2.18.0 (#450186)

r-ryantm committed:
pnpm: 10.18.0 -> 10.18.2

r-ryantm committed:
tutanota-desktop: 309.250918.1 -> 310.251008.0

r-ryantm committed:
home-assistant-custom-lovelace-modules.material-you-utilities: 2.0.14 -> 2.0.16

vbgl committed:
easycrypt: 2025.08 → 2025.10 (#448205)

vbgl committed:
coqPackages.coq-hammer: enable for Coq 9.0 (#449881)

r-ryantm committed:
python3Packages.elevenlabs: 2.16.0 -> 2.17.0

r-ryantm committed:
scx.full: 1.0.16 -> 1.0.17

r-ryantm committed:
jetbrains-toolbox: 2.9.0.56191 -> 2.9.1.58121

r-ryantm committed:
vscode-extensions.svelte.svelte-vscode: 109.11.1 -> 109.11.2

nixpkgs-merge-bot[bot] committed:
odin: dev-2025-09 -> dev-2025-10 (#449751)

nixpkgs-merge-bot[bot] committed:
tmuxai: 1.1.1 -> 1.1.2 (#450490)

r-ryantm committed:
python3Packages.stanza: 1.10.1 -> 1.11.0

r-ryantm committed:
python3Packages.webio-api: 0.1.11 -> 0.1.12

vbgl committed:
rocqPackages.rocq-elpi: 3.1.0 -> 3.2.0 (#450257)

nixpkgs-merge-bot[bot] committed:
amp-cli: 0.0.1759233723-gf92434 -> 0.0.1760013766-g2b1690 (#450334)

nixpkgs-merge-bot[bot] committed:
lianad: 13.0 -> 13.1 (#450415)

nixpkgs-merge-bot[bot] committed:
liana: 13.0 -> 13.1 (#450417)

msanft committed:
aflplusplus: 4.32c -> 4.34c

r-ryantm committed:
vscode-extensions.saoudrizwan.claude-dev: 3.32.5 -> 3.32.7

fabaff committed:
python313Packages.maison: 2.0.0 -> 2.0.2

Changelog: https://github.com/dbatten5/maison/releases/tag/v2.0.2

fabaff committed:
python313Packages.reolink-aio: 0.16.1 -> 0.16.2

Diff: https://github.com/starkillerOG/reolink_aio/compare/0.16.1...0.16.2

Changelog: https://github.com/starkillerOG/reolink_aio/releases/tag/0.16.2

matteo-pacini committed:
jetbrains-toolbox: 2.9.0.56191 -> 2.9.1.58121 (#450552)

hehongbo committed:
xen: patch `libxl` to search for `qemu-system-i386` properly

As upstreamed to Xen Project via `f6281291704aa356489f4bd927cc7348a920bd01`.

Signed-off-by: Hongbo <hehongbo@mail.com>

wegank committed:
msolve: 0.9.1 -> 0.9.2 (#450349)

fabaff committed:
python313Packages.yamlfix: 1.16.1 -> 1.18.0

Changelog: https://github.com/lyz-code/yamlfix/blob/1.18.0/CHANGELOG.md

beeb committed:
foundry: 1.3.0 -> 1.4.0

matteo-pacini committed:
vscode-extensions.saoudrizwan.claude-dev: 3.32.5 -> 3.32.7 (#450574)

fabaff committed:
radare2: 6.0.2 -> 6.0.4 (#450040)

gador committed:
extractpdfmark: link with iconv explicitly on Darwin (#445598)

nixpkgs-merge-bot[bot] committed:
dependency-track: 4.13.4 -> 4.13.5 (#450432)

fabaff committed:
python313Packages.pyexcel: modernize

fabaff committed:
python313Packages.heapdict: modernize

nixpkgs-merge-bot[bot] committed:
laravel: 5.19.0 -> 5.20.0 (#450530)

fabaff committed:
python313Packages.mkdocs-drawio-exporter: modernize

gador committed:
datalad-gooey: apply upstream PR to fix build (#442570)

niklaskorz committed:
shader-slang: 2025.17.2 -> 2025.18.2 (#450298)

ungeskriptet committed:
alglib: fix build with CMake 4

Fixes: https://github.com/NixOS/nixpkgs/issues/450584
Signed-off-by: David Wronek <david.wronek@mainlining.org>

tfc committed:
devenv: 1.9.1 -> 1.10 (#449555)

fabaff committed:
python313Packages.mkdocs-gitlab: modernize

doronbehar committed:
musescore: 4.6.0 -> 4.6.1 (#449941)

leona-ya committed:
transmission_3: fix build with cmake 4 (#449458)

leona-ya committed:
hackrf, soapyhackrf: fix build for cmake 4 (#450306)

fabaff committed:
python3Packages.langsmith: 0.4.31 -> 0.4.34 (#450496)

fabaff committed:
python313Packages.ec2-metadata: update meta

fabaff committed:
python313Packages.ec2-metadata: remove update-script

Removed the nix-update-script from passthru.

leona-ya committed:
libad9361: fix CMake 4 compatibility (#450322)

ungeskriptet committed:
nvidia-texture-tools: fix build with CMake 4

Fixes: https://github.com/NixOS/nixpkgs/issues/450596
Signed-off-by: David Wronek <david.wronek@mainlining.org>

fabaff committed:
python313Packages.optimum: remove disabled

- removed pythonOlder restriction for version 3.9 
- updated the description to remove emoji references

fabaff committed:
python3Packages.gdsfactory: 9.17.0 -> 9.18.1 (#450533)

leona-ya committed:
alglib: fix build with CMake 4 (#450595)

fabaff committed:
python3Packages.elevenlabs: 2.16.0 -> 2.17.0 (#450551)

leona-ya committed:
pamix: fix cmake 4 compatibility (#450449)

fabaff committed:
python313Packages.reolink-aio: 0.16.1 -> 0.16.2 (#450579)

fabaff committed:
python3Packages.ec2-metadata: 2.16.0 -> 2.17.0 (#450391)

fabaff committed:
python313Packages.pyexcel: modernize (#450589)

fabaff committed:
python313Packages.heapdict: modernize (#450588)

nikstur committed:
nixos/timesyncd: cleanup & make it bashless (#449937)

r-ryantm committed:
home-assistant-custom-components.alarmo: 1.10.11 -> 1.10.12

fabaff committed:
python313Packages.maison: 2.0.0 -> 2.0.2, python313Packages.yamlfix: 1.16.1 -> 1.18.0  (#450576)

K900 committed:
vte: fix build on darwin (#450213)

fabaff committed:
checkov: 3.2.474 -> 3.2.479

Diff: https://github.com/bridgecrewio/checkov/compare/3.2.474...3.2.479

Changelog: https://github.com/bridgecrewio/checkov/releases/tag/3.2.479

fabaff committed:
python313Packages.iamdata: 0.1.202510091 -> 0.1.202510101

Diff: https://github.com/cloud-copilot/iam-data-python/compare/v0.1.202510091...v0.1.202510101

Changelog: https://github.com/cloud-copilot/iam-data-python/releases/tag/v0.1.202510101

leona-ya committed:
libbladerf, soapybladerf: fix build for cmake 4 (#450309)

ulrikstrid committed:
microsoft-edge: 141.0.3537.57 -> 141.0.3537.71 (#450428)

fabaff committed:
python3Packages.optimum: 1.27.0 -> 2.0.0 (#450390)

fabaff committed:
python312Packages.mypy-boto3-quicksight: 1.40.46 -> 1.40.49

fabaff committed:
python312Packages.mypy-boto3-wafv2: 1.40.16 -> 1.40.49

fabaff committed:
python313Packages.boto3-stubs: 1.40.48 -> 1.40.49

leona-ya committed:
imgbrd-grabber: fix build with CMake 4 (#450523)

fabaff committed:
python313Packages.mkdocs-drawio-exporter: modernize, python313Packages.mkdocs-gitlab: modernize (#450593)

leona-ya committed:
milkytracker: Fix build with CMake 4 (#450409)

leona-ya committed:
petrifoo: fix cmake 4 compatibility (#450386)

leona-ya committed:
libtins: fix build with cmake 4 (#450277)

r-ryantm committed:
balena-cli: 22.4.9 -> 22.4.10

r-ryantm committed:
libretro.beetle-pce-fast: 0-unstable-2025-09-26 -> 0-unstable-2025-10-03

leona-ya committed:
nvidia-texture-tools: fix build with CMake 4 (#450600)

nixpkgs-merge-bot[bot] committed:
qcal: 0.9.2 -> 0.9.3 (#450508)

nixpkgs-merge-bot[bot] committed:
qcard: 0.7.1 -> 0.7.2 (#450509)

leona-ya committed:
catimg: use proper patch for cmake 4 compatibility (#450222)

thiagokokada committed:
libretro.{citra,dolphin,thepowdertoy,tic80}: fix build (#450251)

r-ryantm committed:
lazyssh: 0.2.1 -> 0.3.0

wineee committed:
lucenepp: fix build with cmake4

fix: https://github.com/NixOS/nixpkgs/issues/450598

markuskowa committed:
OpenOrbitalOptimizer: 0.1.0 -> 0.2.0 (#445777)

matteo-pacini committed:
home-assistant-custom-lovelace-modules.material-you-utilities: 2.0.14 -> 2.0.16 (#450545)

nixpkgs-merge-bot[bot] committed:
fabric-ai: 1.4.308 -> 1.4.319 (#450389)

r-ryantm committed:
mongodb-atlas-cli: 1.48.0 -> 1.49.0

ambroisie committed:
upsies: 2025.09.20 -> 2025.10.09 (#450502)

leona-ya committed:
soapyplutosdr: fix build for cmake 4 (#450312)

leona-ya committed:
soapyremote: fix build for cmake 4 (#450314)

afh committed:
remind: 06.01.02 -> 06.01.04 (#450299)

doronbehar committed:
balena-cli: 22.4.9 -> 22.4.10 (#450614)

matteo-pacini committed:
protoc-gen-grpc-java: 1.75.0 -> 1.76.0 (#450499)

leona-ya committed:
lucenepp: fix build with cmake4 (#450620)

7c6f434c committed:
lincity, lincity-ng: move to by-name, modernize, adopt (#450319)

7c6f434c committed:
xpilot-ng, bloodspilot-{client,server}: move to by-name, modernize (#450247)

r-ryantm committed:
python3Packages.netbox-dns: 1.4.1 -> 1.4.2

LordGrimmauld committed:
opencpn: 5.10.2 -> 5.12.4 (#450273)

nixpkgs-merge-bot[bot] committed:
lazyssh: 0.2.1 -> 0.3.0 (#450618)

LordGrimmauld committed:
libopenglrecorder: fix for CMake v4 (#449888)

r-ryantm committed:
ipxe: 1.21.1-unstable-2025-10-01 -> 1.21.1-unstable-2025-10-09

7c6f434c committed:
abc-verifier: unstable-2023-10-13 → 0.55 (#450176)

nixpkgs-merge-bot[bot] committed:
updatecli: 0.105.0 -> 0.108.0 (#450385)

LordGrimmauld committed:
superTuxKart: fix build against cmake 4 (#449358)

Sigmanificient committed:
vscode-extensions.svelte.svelte-vscode: 109.11.1 -> 109.11.2 (#450557)

Sigmanificient committed:
python3Packages.webio-api: 0.1.11 -> 0.1.12 (#450568)

Sigmanificient committed:
python3Packages.stanza: 1.10.1 -> 1.11.0 (#450566)

fabaff committed:
python312Packages.mypy-boto3-*: updates (#450607)

fabaff committed:
python313Packages.iamdata: 0.1.202510091 -> 0.1.202510101 (#450604)

fabaff committed:
checkov: 3.2.474 -> 3.2.479 (#450603)

thiagokokada committed:
libretro.beetle-pce-fast: 0-unstable-2025-09-26 -> 0-unstable-2025-10-03 (#450615)

SigmaSquadron committed:
ipxe: 1.21.1-unstable-2025-10-01 -> 1.21.1-unstable-2025-10-09 (#450640)

nixpkgs-merge-bot[bot] committed:
mongodb-atlas-cli: 1.48.0 -> 1.49.0 (#450622)

leona-ya committed:
trigger-control: fix build with cmake 4 and other enhancements (#450641)

r-ryantm committed:
python3Packages.siphashc: 2.5 -> 2.6

collares committed:
R: define main program (#450325)

drupol committed:
jiratui: init at 1.2.0 (#450337)

r-ryantm committed:
dependabot-cli: 1.74.0 -> 1.76.0

msanft committed:
foundry: 1.3.0 -> 1.4.0 (#450582)

marcusramberg committed:
vivaldi: 7.6.3797.58 -> 7.6.3797.63 (#450293)

Scrumplex committed:
pnpm: 10.18.0 -> 10.18.2 (#450550)

adamcstephens committed:
audiobookshelf: 2.29.0 -> 2.30.0 (#450513)

r-ryantm committed:
simplex-chat-desktop: 6.4.5 -> 6.4.6

r-ryantm committed:
kdePackages.ktextaddons: 1.7.1 -> 1.8.0

K900 committed:
linux-firmware: 20250917 -> 20251011

Diff: https://gitlab.com/kernel-firmware/linux-firmware/-/compare/20250917...20251011

fabaff committed:
python313Packages.bx-py-utils: disable failing test

K900 committed:
linux-firmware: 20250917 -> 20251011 (#450666)

2hexed committed:
append the extension entry

r-ryantm committed:
mitra: 4.10.1 -> 4.11.0

nixpkgs-merge-bot[bot] committed:
simplex-chat-desktop: 6.4.5 -> 6.4.6 (#450660)

SuperSandro2000 committed:
linuxPackages.openafs: Patch for Linux kernel 6.17 (#449627)

Sigmanificient committed:
openseachest: 25.05.2 -> 25.05.3 (#450522)

erictapen committed:
python3Packages.siphashc: 2.5 -> 2.6 (#450652)

fabaff committed:
python3Packages.netbox-dns: 1.4.1 -> 1.4.2 (#450631)

SuperSandro2000 committed:
oculante: Fix broken build (#449840)

corngood committed:
avalonia: 11.3.6 -> 11.3.7 (#450430)

SuperSandro2000 committed:
ccls: fix build (#450049)

SuperSandro2000 committed:
nautilus-open-any-terminal: 0.6.3 -> 0.7.0 (#450454)

SigmaSquadron committed:
xen: patch `libxl` to search for `qemu-system-i386` properly (#365890)

wentasah committed:
kernelshark: fix build with cmake4

misuzu committed:
foxglove-studio: init at 2.37.0 (#417985)

nixpkgs-merge-bot[bot] committed:
mitra: 4.10.1 -> 4.11.0 (#450674)

mweinelt committed:
home-assistant-custom-components.alarmo: 1.10.11 -> 1.10.12 (#450605)

hsjobeki committed:
nixos/lasuite-meet: make `settings` lazy! (#450327)

prusnak committed:
electrum: fix check without qt (#303916)

SuperSandro2000 committed:
wezterm: 0-unstable-2025-08-14 -> 0-unstable-2025-10-05 (#449928)

r-ryantm committed:
rospo: 0.15.0 -> 0.15.1

leona-ya committed:
kernelshark: fix build with cmake4 (#450681)

r-ryantm committed:
grafana-loki,promtail: 3.5.5 -> 3.5.6

This update includes a security fix for GHSA-2464-8j7c-4cjm.

https://github.com/grafana/loki/releases/tag/v3.5.6

r-ryantm committed:
lxgw-wenkai: 1.520 -> 1.521

philiptaron committed:
dependabot-cli: 1.74.0 -> 1.76.0 (#450656)

r-ryantm committed:
lintspec: 0.9.1 -> 0.10.0

sikmir committed:
rospo: 0.15.0 -> 0.15.1 (#450685)

fabaff committed:
python313Packages.bx-py-utils: disable failing test (#450665)

K900 committed:
kdePackages.ktextaddons: 1.7.1 -> 1.8.0 (#450682)

SchweGELBin committed:
ois: fix cmake 4 compatibility

nixpkgs-merge-bot[bot] committed:
lintspec: 0.9.1 -> 0.10.0 (#450698)

yayayayaka committed:
electron{-source,-bin,-chromedriver}: updates (#449833)

mweinelt committed:
python3Packages.aiohttp-sse: backport patch for pytest-asyncio 1.0 compat

nixpkgs-merge-bot[bot] committed:
tutanota-desktop: 309.250918.1 -> 310.251008.0 (#450543)

yayayayaka committed:
ois: fix cmake 4 compatibility (#449741)

sikmir committed:
reproxy: fix build on x86_64-darwin (#448459)

FliegendeWurst committed:
trilium-{desktop,server}: 0.99.0 -> 0.99.1 (#450261)

emilylange committed:
grafana-loki,promtail: 3.5.5 -> 3.5.6 (#450692)

JohnRTitor committed:
scx.full: 1.0.16 -> 1.0.17 (#450553)

yayayayaka committed:
lightdm-enso-os-greeter: fix build with cmake 4 (#450280)

JohnRTitor committed:
cosmic-greeter: add cosmic-randr to PATH; nixos/cosmic-greeter: install cosmic-comp by default (#448844)

jian-lin committed:
emacs: adjust URL to comment in Emacs’ NEWS file (#450216)

nixpkgs-merge-bot[bot] committed:
lxgw-wenkai: 1.520 -> 1.521 (#450693)

dotlambda committed:
python3Packages.aiohttp-sse: backport patch for pytest-asyncio 1.0 compat (#450703)

matteo-pacini committed:
vscode-extensions.prince781.vala: init at 1.1.0 (#450393)

uninsane committed:
kew: 3.4.1 -> 3.5.2 (#449984)

msanft committed:
aflplusplus: 4.32c -> 4.34c (#450573)

uninsane committed:
pocketbase: 0.30.0 -> 0.30.1 (#449956)

uninsane committed:
ed-odyssey-materials-helper: 3.0.6 -> 3.0.7 (#449959)
